hudi-spark-datasource/hudi-spark3.1.x/pom.xml

```diff
@@ -202,6 +202,18 @@
       </exclusions>
     </dependency>
 
+    <dependency>
+      <groupId>org.apache.hudi</groupId>
+      <artifactId>${hudi.spark.common.module}</artifactId>
```

Comment:
Is this the same dependency as below (`hudi-spark3-common`)?

Reply:
yes this is what is in root pom
```
  <hudi.spark.module>hudi-spark3</hudi.spark.module>
    <hudi.spark.common.module>hudi-spark3-common</hudi.spark.common.module>
```